cpp/include/cudf/detail/aggregation/aggregation.hpp

```diff
@@ -884,6 +890,48 @@ class merge_m2_aggregation final : public groupby_aggregation {
   void finalize(aggregation_finalizer& finalizer) const override { finalizer.visit(*this); }
 };
 
+/**
+ * @brief Derived aggregation class for specifying TDIGEST aggregation
+ */
+class tdigest_aggregation final : public groupby_aggregation {
+ public:
+  explicit tdigest_aggregation(int delta_) : aggregation{TDIGEST}, delta{delta_} {}
+
+  int delta;
```

Comment:
Might we consider `const` here? Should we guard against a publicly modifiable data member?